chore: restored spellcheck

m-kolomanski opened the pull request:
## Issue

Closes #

## Description

Change description.

## Definition of Done

Definition of done, preferably copied from the issue.

## How to test

How to test features not covered by unit tests.

## Contributor checklist
- [ ] Code passes lintr checks
- [ ] Code passes all unit tests
- [ ] New logic covered by unit tests
- [ ] New logic is documented

## Notes to reviewer

Anything that the reviewer should know before tacking the pull request?

Gotfrid commented:
Thank you for making this change, and confirming that it works!

Considering that #146 is still in review, let's wait - maybe we get some meaningful suggestions  in [pharmaverse/admiralci](https://github.com/pharmaverse/admiralci/issues/198).

If #146 is ready before we get an answer, then let's proceed as we discussed - copy over the workflow files, modify them to install dependencies, and reference them from the main workflow.

m-kolomanski commented:
@Gotfrid FYI, I have implemented the discussed workflow structure (copied files over, added minimal modifications, referenced in main file) and verified that everything works correctly with `units` package, the dependencies are being installed.

Gotfrid reviewed (approved):
LGTM, thanks!

m-kolomanski commented:
It does not look like we are going to get any meaningful answer. I am merging the PR. If any better solution arises, we can go back to this issue.
